src/main/resources/snippets/jython/filter-map-reduce.py

```diff
@@ -0,0 +1,4 @@
+def run(data):
+    return reduce(lambda acc, next: acc + next,
+        map(lambda x: x * 2,
+            filter(lambda x: x % 2 == 0, data)), 0)
```

Comment:
maybe we could here use the operator module...
```python
import operator
from functools import partial

def run(data):
  return reduce(operator.add, map(partial(operator.mul, 2), filter(lambda x: x % 2 == 0, data)), 0)
```

depending on what you want to test